DEPLOYMENT_USAGE.MD

```diff
@@ -0,0 +1,187 @@
+      "airseekerConfig": { "deviationThreshold": 1, "heartbeatInterval": 86400, "updateInterval": 30 }
+    }
+    "bsc": {
+      "active": true,
+      "sponsor": "0xAb770D6E94e26.......",
+      "topUpWallets": [],
+      "updateConditionPercentage": 2,
+      "airseekerConfig": { "deviationThreshold": 1, "heartbeatInterval": 86400, "updateInterval": 30 }
+    }
+  }
+}
+```
+
+`beaconId` and `topUpWallets` can be empty when populating the beacon data.
+
+`updateConditionPercentage` is the deviation at which the beacon will be updated by airkeeper (ran by the provider). Absence of this field means that the beacon is not included in the `airkeeper.json` configuration for the specified chain.
+
+`airseekerConfig` is the configuration of airseeker (ran by API3). Absence of this field means that the beacon is not included in the `airseeker.json` configuration for the specified chain.
+
+> Note that the above files can have any name initially as the validator will name them according to their `beaconName`.
+
+Once populated run the following command to generate the `Provider-Sponsor` TopUp Wallet used by airkeeper
+
+```
+yarn generate-topup-wallets
+```
+
+> There is also a prompt to specify the `API3` TopUp Wallet which is used by airseeker:
+
+When complete run the following command to normalize the repository (this will also generate the beaconIds).
+
+```
+yarn normalize
+```
+
+### 6. Generate the Deployment files
+
+run the following command to create a set of config files for deployment
+
+```
+yarn create-config
+```
+
+This will create following files for both the airnode and airkeeper deployment in `/data/apis/<PROVIDER_NAME>/deployments/<DEPLOYMENT_DATE>`
+
+- `airnode`
+  - `config.json`: The Airnode config file, should not require editing.
+  - `secrets.env`: This file will need to be populated by the provider during deployment.
+  - `aws.env`: This file will need to be populated by the provider during deployment
+- `airkeeper`
+  - `config.json`: The Airnode config file, should not require editing.
+  - `airkeeper.json`: The Airkeeper config file, should not require editing.
+  - `secrets.env`: This file will need to be populated by the provider during deployment.
+  - `aws.env`: This file will need to be populated by the provider during deployment
+
+### 7. Generate the gateway keys
+
+Use the following command to generate the gateway keys
+
+```
+yarn generate-gateway-key
+```
+
+These keys need to be stored on the API3 keybase account and will be forwarded to the API provider during deployment to be filled in `secrets.env` .
+
+### 8. Checkout and publish
+
+Once you have populated the config files, avoiding populating them with any secrets.
+
+Checkout using `git checkout -b ...`. Use a new descriptively-named branch, eg. "nasa-deployment" and push the branch to the operations repository.
+
+### 9. Deployment at the client's premises:
+
+- Clone the repository and switch to the earlier-created branch
```

Comment:
- Clone the repository and switch to the branch created in previous step (or step 8)?